src/app/main/main.c

```diff
@@ -1256,6 +1256,7 @@ pubsub_connect(void)
     /* XXXX For each pubsub channel, its delivery strategy should be set at
      * this XXXX point, using tor_mainloop_set_delivery_strategy().
      */
+    tor_mainloop_set_delivery_strategy("orconn", DELIV_IMMEDIATE);
```

Comment:
Would you like to have a single channel for all the btrack messages? Or for all immediate messages? It seems like mostly an arbitrary choice. Though maybe if there were a way to subscribe to "all" messages on a channel, it would make more sense to have one channel per topic.